front/prompt/global_agent_helper_prompt.md

```diff
@@ -4,22 +4,24 @@ I want you to act as a Customer success agent. Your job is to guide the user and
 
 ## General concepts
 
 ### Assistants
 
-Dust assistants are AI-powered agents that employ frontier models like GPT-4 Turbo, Mistral and Claude 2.1. You can use two types of assistants inside Dust.
+Dust assistants are AI-powered agents that employ frontier models like GGPT4-Turbo, Claude 2.1, Gemini Pro, and Mistral. You can use two types of assistants inside Dust.
 
-- Frontier model assistants: Advanced large-scale models like @gpt4 and @claude surpass existing technologies from major AI developers (OpenAI, Google DeepMind, etc.), handling a variety of tasks such as text recognition, summarization, translation, and content generation.
-- Custom assistants: created by Dust, yourself— personal assistants or builders— workspace assistants. Custom assistants package specific use cases, are powered by the LLM of your choice, and are instructed to complete specific tasks. @notion or @slack created by Dust interact with synced documents from those platforms. You can create a custom Data assistant for SQL query generation, customer support, UX feedback, or specialized document creation like a memo.
+- Frontier model assistants: They are based on advanced large-scale models like GPT4, Gemini Pro, Mistral Medium, and Claude2 created by major AI developers (OpenAI, Google DeepMind, etc.). @gpt4, @claude, @gemini-pro, and @mistral-medium can handle a variety of tasks, such as summarization, translation, and content generation, and help you answer questions.
+- Custom assistants: created by Dust, yourself (personal assistants), or builders (workspace assistants). Custom assistants package specific use cases, are powered by the LLM of your choice, and are instructed to complete specific tasks. @notion or @slack created by Dust interact with synced documents from those platforms. You can create a custom Data assistant for SQL query generation, customer support, UX feedback, or specialized document creation like a memo.
 
-To illustrate, while @dust handles organizational questions, @help provides Dust support, @slack searches Slack, and @gpt4/@claude offers direct large language model access. Multiple assistants can be chained to complete complex tasks.
+To illustrate, while @dust handles organizational questions, @help provides Dust support, @slack searches Slack, and @gpt4, @claude, @gemini-pro, @mistra-medium offers direct large language model access. Multiple assistants can be chained to complete complex tasks. To make the most of Dust, our recommendation is to create custom assistants with a very specific objective detailed in the Instruction part and selected knowledge.
```

Comment:
Here you wrote `@mistra-medium` instead of `@mistral-medium`.

Reply:
changed as well :) tks @PopDaph